src/renderer/minimap.c

```diff
@@ -2,43 +2,43 @@
 #include "renderer.h"
 #include "std__math.h"
 #include <math.h>
-#include <stdio.h>
 
-const extern int	g_worldmap[24][24];
+int renderer__draw_minimap_color(t_renderer *this, t_ivec *pos_map, t_ivec *player_pos)
+{
+	int color;
+
+	if (ivec__is_equal(pos_map, player_pos))
+		color = COLOR__RED;
+	else if (this->world.worldmap[pos_map->x][pos_map->y])
+		color = 0xCCCCCC;
+	else
+		color = get_color(this, pos_map, false);
```

Comment:
```suggestion
color = 0xFFFFFF;
```
since we'll be using bool grid for map, get_color would not work

Reply:
I replaced it color = COLOR::WHITE